furusiya/model/entities/party/player.py

```diff
@@ -89,14 +91,16 @@ def move_or_attack(self, dx, dy):
             Game.renderer.recompute_fov = True
             return
 
-        self.get_component(Fighter).attack(target)
+        FighterSystem.get_fighter(self).attack(target)
+        if config.data.skills.omnislash.enabled:
+            OmniSlash.process(self, config.data.skills.omnislash.rehitPercent, (dx, dy))
```

Comment:
Why do you need `dx, dy` here?

Reply:
My bad, I see it. There's a chance you'll move over the monster's corpse if it dies, right?

Reply:
Yep, and you can see it in action if you set the rehit chance to 100%.
But you know,
![its_not_a_bug_its_a_feature](https://user-images.githubusercontent.com/26042759/36274214-13f92d86-1298-11e8-9a5c-c03fd342a5c3.jpg)